editor.Rmd

```diff
@@ -149,52 +147,54 @@ Tag the account(s) of all post authors and package authors and reviewers.
+For a post about a package, make the package name a hashtag.  You can also add the package category, as we use it for social media campaigns. For example, for a post about the `rredlist` package, you could use `#rredlist` and `#biodiversity`.
 
-Check the last tweets using any hashtag except the `#rstats` hashtag to see whether it is used as you expect.
-E.g. using `httr` wouldn't be a good idea. It is a package name but on Twitter it is the hashtag of [a team with a name controversy](https://en.wikipedia.org/wiki/Washington_Redskins_name_controversy) and even has its own emoji automatically attached to it by Twitter. 
+Check the last post using any hashtag except the `#rStats` hashtag to see whether it is used as you expect.
+E.g. using `httr` wouldn't be a good idea. It is a package name but on social media it is the hashtag of [a team with a name controversy](https://en.wikipedia.org/wiki/Washington_Redskins_name_controversy). 
 
 #### Adding hashtags
 
-Do not use too many hashtags in any tweet so as not to make the account look like a spam account / greedy for attention.
+Do not use too many hashtags in any post so as not to make the account look like a spam account / greedy for attention.
 
-Post hashtags at the bottom of each tweet, to make the rest of the content of the tweet easier to read.
+Post hashtags at the bottom of each post, to make the rest of the content of the post easier to read.
 
 [Capitalize letters of each word](https://www.boia.org/blog/make-your-hashtags-accessible) for hashtags including several words i.e. use lower or upper camelCase (e.g. `#RLadies` not `#rladies`). 
 
 When posting a thread, use each hashtag only once so as not to pollute the timeline of that hashtag.
 
 ### Using emojis 
 
-In tweets, emojis are optional. 
+In post, emojis are optional. 
 When using emojis, [do not use too many of them](https://www.a11ywithlindsey.com/blog/7-ways-tweets-more-accessible).
 
 ### Using gifs
 
-In tweets, gifs are optional.
+In post, gifs are optional.
 
 * One can include a gif that was created for the post (e.g. an animated plot).
 * But generally not a Giphy gif since it might be interpreted differently by people who know more/less context about the gif in question (e.g. a TV show).
-* When using a gif we should [add its description](https://www.a11ywithlindsey.com/blog/7-ways-tweets-more-accessible) by typing `[Gif alt: descriptive phrase]` at the bottom of the tweet.
-* When tweeting from twitter.com, as of Feb 2020, if you have the _compose image descriptions_ setting turned on in your accessibility settings, you can add alt descriptions for gifs as you do for images.
+* When using a gif we should [add its description](https://www.a11ywithlindsey.com/blog/7-ways-tweets-more-accessible) by typing `[Gif alt: descriptive phrase]` at the bottom of the post.
```

Comment:
I also always wonder what the copyright implications of using gifs, from movies for instance, are.

Reply:
I added this comment in an issue so we can take it in the future and not delay the merge of all these changes.